app/controllers/auth_controller.py

```diff
@@ -0,0 +1,71 @@
+from flask import request, jsonify, Blueprint
+
+from app.exceptions import InvalidRequestFieldError, EmailTakenError, InvalidPasswordError, UserNotExistError
+from app.reponses import ApiErrorResponse
+from app.requests import RegisterRequest, LoginRequest
+from app.services.auth_service import AuthService
+
+
+class AuthController:
+    def __init__(self, auth_service: AuthService):
+        self.auth_service = auth_service
+
+    @staticmethod
+    def is_valid_field(field):
+        return field is not None and field.strip() != ""
+
+    @staticmethod
+    def _verify_and_prepare_register_request() -> RegisterRequest:
+        request_data = request.get_json()
+        if request_data is None:
+            raise InvalidRequestFieldError
+
+        name = request_data.get('name')
+        email = request_data.get('email')
+        password = request_data.get('password')
+
+        if (
+                not AuthController.is_valid_field(name)
+                or not AuthController.is_valid_field(email)
+                or not AuthController.is_valid_field(password)
+        ):
+            raise InvalidRequestFieldError
+
+        return RegisterRequest(
+            name=name,
+            email=email,
+            password=password)
```

Comment:
Nie ma jakiegos skrotu na zapis tego? Tak dziwnie to wyglada xd

Reply:
nie rozumiem, jak to powinno wyglądać? może być w jednej linijce ewentualnie

Reply:
zwinąłem w jedną linijke